external-crates/move/crates/move-analyzer/editors/code/src/context.ts

```diff
@@ -42,9 +42,9 @@ function semanticVersion(versionString: string | null): semver.SemVer | null {
 }
 
 function shouldInstall(bundledVersionString: string | null,
-                       bundledVersion: semver.SemVer | null,
-                       highestVersionString: string | null,
-                       highestVersion: semver.SemVer | null): boolean {
+    bundledVersion: semver.SemVer | null,
```

Comment:
Sorry for these. Apparently I never ran a formatter on this file (or maybe edited these at some point in the non-auto-formatted way...)